tools/SourceKit/lib/SwiftLang/SwiftDocSupport.cpp

```diff
@@ -1366,7 +1366,7 @@ SourceFile *SwiftLangSupport::getSyntacticSourceFile(
     Error = "Compiler invocation set up failed";
     return nullptr;
   }
-  ParseCI.performParseOnly();
+  ParseCI.performParseOnly(/*EvaluateConditionals*/true);
```

Comment:
@benlangmuir Is this correct?  Should this be a parameter to this function?  Its callers are the 'syntactic rename' and the 'find rename ranges' actions.

Reply:
Seems like a fine stopgap to me.